README.md

```diff
@@ -7,6 +7,29 @@ An opinionated static site generator written in Rust.
 Documentation is available on [its site](https://www.getgutenberg.io/documentation/getting-started/installation/) or
 in the `docs/content` folder of the repository.
 
+## Comparisons with other static site generators
+
+|                          | Gutenberg | Cobalt | Hugo | Pelican |
+|--------------------------|-----------|--------|------|---------|
+| Single binary            |     ✔     |    ✔   |   ✔  |    ✕    |
+| Language                 |    Rust   |  Rust  |  Go  |  Python |
+| Syntax highlighting      |     ✔     |    ✔   |   ✔  |    ✔    |
+| Sass compilation         |     ✔     |    ✔   |   ✕  |    ✔    |
+| Assets co-location       |     ✔     |    ✕   |   ✔  |    ✔    |
+| i18n                     |     ✕     |    ✕   |   ✔  |    ✔    |
+| Image processing         |     ✕     |    ✕   |   ✔  |    ✔    |
+| Search                   |     ✕     |    ✕   |   ✕  |    ✔    |
+| Sane template engine     |     ✔     |    ✔   |  ✕✕✕ |    ✔    |
+| Themes                   |     ✔     |    ✕   |   ✔  |    ✔    |
+| Shortcodes               |     ✔     |    ✕   |   ✔  |    ✔    |
+| Internal links           |     ✔     |    ✕   |   ✔  |    ✔    |
+| Table of contents        |     ✔     |    ✕   |   ✔  |    ✔    |
+| Automatic header anchors |     ✔     |    ✕   |   ✔  |    ✔    |
+| Aliases                  |     ✔     |    ✕   |   ✔  |    ✔    |
+| Pagination               |     ✔     |    ✕   |   ✔  |    ✔    |
+| Custom taxonomies        |     ✕     |    ✕   |   ✔  |    ✕    |
```

Comment:
@epage anything I got wrong for Cobalt or a feature I missed?

Reply:
In case you are interested in linking to the issues
- https://github.com/cobalt-org/cobalt.rs/issues/395 for pagination / ToC
- https://github.com/cobalt-org/cobalt.rs/issues/392 for i18n
- https://github.com/cobalt-org/cobalt.rs/issues/301 for themes
- https://github.com/cobalt-org/cobalt.rs/issues/368 for custom taxonomies (if I understand that Hugo feature correctly)

Also could you clarify something for me.  I assume the only place worth doing pagination is in a ToC, so why are they listed separately?

Reply:
- https://github.com/cobalt-org/cobalt.rs/issues/318 and https://github.com/cobalt-org/cobalt.rs/issues/321 for image processing though I was looking at it from a different angle than Hugo.

Reply:
I'm not going to link to issues to keep it short.

> I assume the only place worth doing pagination is in a ToC, so why are they listed separately?

Hmm why would have pagination in a ToC? A ToC is like what I'm using for the left menu of https://tera.netlify.com/docs/installation/ for example. 
Pagination is for pages like a blog or sometimes for taxonomies like the `tags/some-tag` list pages where you list all pages.

Reply:
Ok, didn't look too closely at what a ToC is and assumed it was an index.